patches/node/fix_crypto_tests_to_run_with_bssl.patch

```diff
@@ -471,10 +471,21 @@ index 008ab129f0e019c659eecf5a76b7eb412c947fe3..6688f5d916f50e1e4fcfff1619c8634a
  
  cipher.end('Papaya!');  // Should not cause an unhandled exception.
 diff --git a/test/parallel/test-crypto-x509.js b/test/parallel/test-crypto-x509.js
-index 89a7521544f7051edc1779138551bbad1972b3fb..91df6acc65d4003999f29f0fa5f639056b21ee3b 100644
+index 15e1f53bb05faf60fa808eb5901591d1512edf3c..b04b574b5e683cdddb3836972e246df99e73ac34 100644
 --- a/test/parallel/test-crypto-x509.js
 +++ b/test/parallel/test-crypto-x509.js
-@@ -111,7 +111,7 @@ const der = Buffer.from(
+@@ -96,8 +96,8 @@ const der = Buffer.from(
+   assert.strictEqual(x509.infoAccess, infoAccessCheck);
+   assert.strictEqual(x509.validFrom, 'Sep  3 21:40:37 2022 GMT');
+   assert.strictEqual(x509.validTo, 'Jun 17 21:40:37 2296 GMT');
+-  assert.deepStrictEqual(x509.validFromDate, new Date('2022-09-03T21:40:37Z'));
+-  assert.deepStrictEqual(x509.validToDate, new Date('2296-06-17T21:40:37Z'));
++  // assert.deepStrictEqual(x509.validFromDate, new Date('2022-09-03T21:40:37Z'));
++  // assert.deepStrictEqual(x509.validToDate, new Date('2296-06-17T21:40:37Z'));
```

Comment:
I'm pretty unfamiliar with this code, but is there a condition we could gate this on instead of commenting it out? Similar to where we have checks for `common.openSSLIsBoringSSL`. If not, that's fine too.

Reply:
yeah, we can use `common.openSSLIsBoringSSL` here too!